packages/api/src/Analytics.ts

```diff
@@ -1,7 +1,7 @@
 import Segment from 'analytics-node';
```

Comment:
Not sure if `Analytics` should be in the `@expo/api` package. I think it makes sense to keep libraries centered around a single purpose. Analytics doesn't interact with the expo.io API and also doesn't have strong interdependencies with this library. Maybe throw it into a separate package or move to `expo-cli`?

Reply:
Can't put it in expo-cli because expo-cli already deps expo/api. Makes sense to split it out but that might be a bit heavy for the single module, prolly want to audit the analytics and see if we can reduce the surface area. We need the api package pretty quickly to get notifications and auth-session working for 41 so it seems reasonable it leave Analytics in api for now and move it later.

Reply:
Related https://github.com/expo/expo-cli/pull/3372#discussion_r609405158